csharp/BUILDING.md

```diff
@@ -3,90 +3,48 @@
 This page describes how to build Ice for .NET from source and package the resulting binaries. As an alternative, you
 can download and install the [zeroc.ice.net][1] NuGet package.
 
-* [Building on Windows](#building-on-windows)
-  * [Windows Build Requirements](#windows-build-requirements)
-  * [Compiling Ice for \.NET on Windows](#compiling-ice-for-net-on-windows)
-    * [Authenticode Signatures](#authenticode-signatures)
-* [Building on Linux or macOS](#building-on-linux-or-macos)
-  * [Linux and macOS Build Requirements](#linux-and-macos-build-requirements)
-  * [Compiling Ice for \.NET on Linux or macOS](#compiling-ice-for-net-on-linux-or-macos)
+* [Building](#building)
+  * [Prerequisites](#prerequisites)
+  * [Compiling Ice for \.NET](#compiling-ice-for-net)
+  * [Authenticode Signatures](#authenticode-signatures)
 * [Running the Tests](#running-the-tests)
 * [NuGet Package](#nuget-package)
 
-## Building on Windows
+## Building
 
-### Windows Build Requirements
+### Prerequisites
 
-In order to build Ice for .NET from source, you need:
+1. .NET SDK 8.0 \
+Download the .NET SDK from [dotnet.microsoft.com](https://dotnet.microsoft.com/en-us/download/dotnet).
 
-* Visual Studio 2022 with [.NET 8.0 SDK][2].
+2. The Slice to C# compiler from the C++ source distribution. \
+Refer to the [build instructions](../cpp/BUILDING.md) in the `cpp` folder for details on how to build the C++ source distribution.
 
-### Compiling Ice for .NET on Windows
+3. Python 3.12 is required for running the tests. The Glacier2 test additional require the passlib Python package.
 
-Open a Visual Studio command prompt and change to the `csharp` subdirectory:
+### Compiling Ice for .NET
 
-```shell
-cd csharp
-```
-
-To build all Ice assemblies and the associated test suite, run:
+Open a command prompt and change to the `csharp` subdirectory. To build all Ice assemblies and the associated
+test suite, run:
 
 ```shell
-msbuild msbuild\ice.proj
+dotnet msbuild msbuild\ice.proj
```

Comment:
We should decide whether (a) building ice/csharp builds slice2cs or (b) requires slice2cs to be previously built in ice/cpp/bin/...

If (a), then my understanding is we could / should use `dotnet build` for these instructions. If (b), then we don't need "2" in the prerequisites.

Reply:
On Windows we were using `MSBuild` command to be able to build `slice2cs`. We cannot use `dotnet msbuild` to build C++ binaries.

We are not using `dotnet build` because our NuGet package `zeroc.ice.net` doesn't match any of the projects we build it just aggregate all the other projects in a single package.